src/main/java/uk/gov/hmcts/sptribs/caseworker/model/RecordListing.java

```diff
@@ -183,12 +183,12 @@ public class RecordListing {
 
     @JsonIgnore
     public String getSelectedRegionVal() {
-        return this.getRegionList().getValue().getLabel();
+        return this.getRegionList() != null ? this.getRegionList().getValue().getLabel() : null;
     }
```

Comment:
Could use Objects.nonNull here? Also, for consistency, if you're null checking `this.getRegionList()`, should you not null check `this.getRegionList().getValue()`?

Reply:
this.getRegionList().getValue() can never be null too. We are giving default label as expected by DynamicList. So this check is not really needed.